examples/Gmail-Connector/README.md

```diff
@@ -0,0 +1,151 @@
+    }
+};
+
+gmail:Client gmailClient = new(gmailConfig);
+```
+
+## How it works
+In this example, we make use of the GMail connector to send a confirmation email regarding the payment status of a doctor's appointment.
+
+
+A JSON payload containing the necessary details is sent to the service. 
+
+```json
+{
+"name": "John Doe",
+"dob": "1940-03-19",
+"ssn": "234-23-525",
+"address": "California",
+"phone": "8770586755",
+"email": "johndoe@gmail.com",
+"doctor": "thomas collins",
+"hospital": "grand oak community hospital",
+"cardNo": "7844481124110331",
+"appointment_date": "2025-04-02"
+}
+```
+
+ A request is made to the `Hospital-Service` which returns a JSON payload with the confirmation details such as the appointment number and fees. 
+ 
+ This payload can be used to generate a professional looking email, but for now we will just be sending it unaltered
+
+```json 
+{ 
+   "appointmentNo":1,
+   "doctorName":"thomas collins",
+   "patient":"John Doe",
+   "actualFee":7000.0,
+   "discount":20,
+   "discounted":5600.0,
+   "paymentID":"480fead2-e592-4791-941a-690ad1363802",
+   "status":"Settled"
+}
+```
+With the necessary `MessageRequest` details specified, the payload can be sent to its intended recipient and a reponse is sent back to the caller confirming the receipt of the email. 
+
+```json
+{
+    "Message": "The email has been successfully sent",
+    "Recipient": "someone@gmail.com"
+}
+```
+
+## More on Ballerina's Gmail Module
+The `sendMessage` remote function sends an email. `MessageRequest` is an object that contains all the data that is required
+to send an email. The `userId` represents the authenticated user and can be a Gmail address or ‘me’ 
+(the currently authenticated user).
+
+```ballerina
+string userId = "me";
+gmail:MessageRequest messageRequest = {};
+messageRequest.recipient = "recipient@mail.com";
+messageRequest.sender = "sender@mail.com";
+messageRequest.cc = "cc@mail.com";
+messageRequest.subject = "Email-Subject";
+messageRequest.messageBody = "Email Message Body Text";
+//Set the content type of the mail as TEXT_PLAIN or TEXT_HTML.
+messageRequest.contentType = gmail:TEXT_PLAIN;
+//Send the message.
+var sendMessageResponse = gmailClient->sendMessage(userId, messageRequest);
+```
+
+The response from `sendMessage` is either a string tuple with the message ID and thread ID 
+(if the message was sent successfully) or an `error` (if sending the message was unsuccessful).
+
+```ballerina
+if (sendMessageResponse is (string, string)) {
+    // If successful, print the message ID and thread ID.
+    (string, string) (messageId, threadId) = sendMessageResponse;
+    io:println("Sent Message ID: " + messageId);
+    io:println("Sent Thread ID: " + threadId);
+} else {
+    // If unsuccessful, print the error returned.
+    io:println("Error: ", sendMessageResponse);
+}
+```
+
+The `readMessage` remote function reads messages. It returns the `Message` object when successful or an `error` when unsuccessful.
```

Comment:
Details on reading and deleting should be under Gmail Module example. We should point to that doc from this doc saying something similar to **details on Gmail module can be found at ...**

Reply:
Done